setup/ipc-meta-setup.sh

```diff
@@ -70,8 +78,16 @@ ls -1 "${BASEDIR}"/[0-9]*.sh | sort | while read SCRIPT; do
         [ "$EVERY_TIME" = "yes" ] || continue
     fi
 
-    echo "APPLY: running ${SCRIPT_NAME}..."
-    ${SCRIPT} || die "${SCRIPT} failed with exit-code $?, not proceeding with other scripts"
-    touch "${SETUPDIR}/${SCRIPT_NAME}.done"
+    echo "APPLY: running ${SCRIPT_NAME} at `date -u`..."
+    if [ -x "${SCRIPT}" ]; then
+        ${SCRIPT}
```

Comment:
Shouldn't that be surrounded by quotes?

Reply:
possibly... so far I kept that as it was in the originally working state (which matched the file names we have deployed), but yes better safe than sorry :)